### DISCUSSION TITLE
Request to add sequelizemm to resource page

### DISCUSSION THREAD

**Discussion by hasinoorit** [sequelize/sequelize#15894] [Help & Questions] [answered]
Hi, I want to add, `sequelizemm` library to the resource page of the documentation website. I have developed this library for generating migration script based on model changes. I am using this library for few month, Today i decided to make it public. you can check it at https://github.com/hasinoorit/sequelizemm. Thanks for your feedback.

**Comment by ephys** [comment, marked as answer]
Hi :)

Sure! Feel free to send a PR our way. You can do so easily by clicking the edit button at the bottom of this page: https://sequelize.org/docs/v6/other-topics/resources/ & this one https://sequelize.org/docs/v7/other-topics/resources/

**Reply by hasinoorit** [reply]
Thanks, I have make a PR. [#455](https://github.com/sequelize/website/pull/455). Please check and merge.
